src/VisualStudio/Setup.Next/VisualStudioSetup.Next.csproj

```diff
@@ -35,6 +35,10 @@
       <IncludeOutputGroupsInVSIX>BuiltProjectOutputGroup%3b</IncludeOutputGroupsInVSIX>
       <IncludeOutputGroupsInVSIXLocalOnly>DebugSymbolsProjectOutputGroup%3b</IncludeOutputGroupsInVSIXLocalOnly>
     </ProjectReference>
+    <ProjectReference Include="..\Core\Def.Next\ServicesVisualStudio.Next.csproj">
+      <Project>{d8c86336-f959-47f9-9f26-2725b4bc79ff}</Project>
+      <Name>ServicesVisualStudio.Next</Name>
```

Comment:
@Pilchie can you clarify this comment?  I included the same items that VisualStudioSetup.csproj has.  What do you think i should do here?  Thanks!